app/views/requests/_letter_of_consent.html.erb

```diff
@@ -1,5 +1,14 @@
-<%= f.govuk_file_field :letter_of_consent, label: nil, hint: { text: simple_format(t("helpers.hint.request_form.letter_of_consent")) } %>
-<p class="govuk-body-m"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
+<span class="govuk-body">You must provide a letter of consent from the subject of this request which:</span>
+<ul class="govuk-list govuk-list--bullet" style="margin-bottom:0;">
+  <li>States that they consent to their information being released to you</li>
+  <li>Is signed by them</li>
+  <li>Is dated within the last 6 months</li>
+</ul>
```

Comment:
Please use govuk_list component for lists.

Reply:
I thought I was. I've copied below:

<p class="govuk-body">You can buy:</p>
<ul class="govuk-list govuk-list--bullet">
  <li>apples</li>
  <li>oranges</li>
  <li>pears</li>
</ul>

Reply:
I can see you have copied the output from the help page. You need to look at the input and use the component which will then produce that HTML.

`<%= govuk_list ["apples", "oranges", "pears"], type: :bullet %>`